packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -84,6 +87,9 @@ export const replayCommandHandler: (
   cookies,
   cookiesFile,
 }) => {
+  let closeServerIfRunning = () => {
```

Comment:
`let` :scream: 

Also, no need to define a noop function. If `closeServerIfRunning` has type `Function | null | undefined`, you can call it with `closeServerIfRunning?.();` (will call the function if it is not null or undefined).